### ISSUE TITLE
Final project reviews

### ISSUE THREAD

**Issue by syahrulhamdani** [IdhamHabibie/H8_PTP_Final_Image#1]
Nice! Good job on finishing the project. However, there are some improvements you may want to add to your notebook.
1. You may want to add some introduction to the first markdown cell.
```
## Cat and Dog Classifications
###  Name: Muhammad Idham Habibie
###  Student Number: PYIM004UH004

Your introductions here (abstract, project objective, project background, etc.)
```
2. IMO, you don't need to print the image and labels (at `In [5]:`) since it prints a lot of arrays. Or, if you insist, you can use `trainDataset.take(1)` to only sample 1 image and label. Still, it looks ugly to print a lot of arrays.
3. At `4. Deploy the Convolutional Neural Network (CNN)`, it should be `4. Define The Model ...` or something, we don't do any deployment here yet.
4. You may want to print the final accuracy and losses for the training and validation set, right after fitting the model (by doing `model.evaluate` on both training and validation set).
5. Looks like your model doesn't do a pretty good job, you may consider researching different architecture to improve the performance.
6. Weird. The accuracy is around `0.5` but the model predicted all images correctly. How is that?
